### PR TITLE
Removed edit button from wiki layout file

### LINKED ISSUE #574: Remove edit button from wiki layout file
That’s an issue for newcomers! Feel free to work on it, if you want to. 

**Context**

https://github.com/jamulussoftware/jamuluswebsite/blob/084aeaba2aee8f456d07bbe5e8f531a304298a13/_layouts/wiki.html#L18
**Background**
Since the new process will no longer have .md files the repo, there’s no need to have an edit button on the page. Also I doubt many people are using it anyway.

**Todo**
—> Remove lines 18 to 26 in the file linked above
—> Search for and remove #editsitecontent rules in all .css files in /assets/css/
—> Remove the "githubrepoeditbase" entry in the _config.yml file
—> Open a pull request and await a review

### PR DESCRIPTION
**Pull request by DevRish**
# Changes

Removed lines 18 to 26 in wiki.html.
Removed #editsitecontent rules in all .css files in /assets/css/
Removed the "githubrepoeditbase" entry in the _config.yml file
Removed all occurances of btnEditAlt in _data/*/general.yml

### CODE REVIEW

**Review by ann0see — COMMENTED** [review]
Great! Thank you very much. 

I‘ll test it locally and merge it if everything is ok.

File: _layouts/wiki.html [hunk 1]
```diff
@@ -15,15 +15,6 @@
     <div id="maincontainer" class="fx-col-100-xs fx-col-50-m">
       <noscript class="red"><strong>{{ site.data.general.tNoJSEnabled }}</strong></noscript>
       <main lang="{{ page.lang }}">{{ content }}</main>
-      <div id="editsitecontent">
-        <a href="{{ site.githubrepoeditbase }}/{{ page.path }}" target="_blank" rel="noopener noreferrer">
-          <svg aria-labelledby="etitle" width="20" height="20" viewBox="0 0 1792 1792" xmlns="http://www.w3.org/2000/svg">
-            <title id="etitle">{{ site.data.general.btnEditAlt }}</title>
```
Comment (ann0see) [inline _layouts/wiki.html:21]: Sorry, I must have missed that in #574: You should also remove all the btnEditAlt entries in _data/*/general.yml
That's my fault.

**Review by ann0see — CHANGES_REQUESTED** [review]

File: _layouts/wiki.html (hunk 1 above)
Comment (DevRish) [reply _layouts/wiki.html:21]: Ok I will remove them too.

Comment (DevRish) [reply _layouts/wiki.html:21]: Done

Comment (ann0see) [reply _layouts/wiki.html:21]: Thanks!

**Review by ann0see — APPROVED** [review]
Thank you for working on this. Going to merge it.

**Pull request merged**